06-building-a-program-start-to-finish/automating-tasks/output/README.md

```diff
@@ -0,0 +1,5 @@
+#### Output
+  - [Automating task screencast](https://www.youtube.com/watch?v=QhK69ChDRg4&feature=youtu.be)
```

Comment:
I have done another screencast. This time showing how gulp refreshes my html, js and css files.